drud/ddev
# Note that files_pull_command should not be empty

## The Problem/Issue/Bug:

the docs here are slightly deceptive. `files_pull_command` is noted as required but then as optional.

## How this PR Solves The Problem:

Clarify that `files_pull_command` is required, but can just be a stub.

## Manual Testing Instructions:

- Make a new provider
- Comment out or delete `files_pull_command`
- Run `ddev pull provider` and observe `No FilesPullCommand is defined for provider ...`
- Insert a command that returns true:
  ```
  files_pull_command:
    command: |
      true
    service: web```
- run `ddev pull provider` and observe success

## Automated Testing Overview:


## Related Issue Link(s):

## Release/Deployment notes:




<a href="https://gitpod.io/#https://github.com/drud/ddev/pull/3871"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
